### DISCUSSION TITLE
手机端显示的问题

### DISCUSSION THREAD

**Discussion by zqm971210** [zxuqian/frontend-questions#5] [求助]
## 我做的小项目在手机端显示的时候出现了一个问题？

*up你好 这是我最近在做的小项目(http://www.ehbio.com/Ecker/dist/#/) 第一次做,做的不是很好,目前我遇到一个问题, 我想把我做的项目对手机进行适配,当我在浏览器的中打开项目的时候,出现下面的问题*
![image](https://user-images.githubusercontent.com/48125795/96841761-c2b4dd80-147e-11eb-87e8-dd2a3cb36668.png)

## 我尝试过设置 这个div的宽度 这个边框就会消失 

![image](https://user-images.githubusercontent.com/48125795/96841966-090a3c80-147f-11eb-9a3c-c970a8d99828.png)

**Comment by zxuqian** [comment]
是这个邮箱的问题，文字超长了：
![image](https://user-images.githubusercontent.com/2687921/97007826-91b6d480-1574-11eb-87b7-6ffba84b0fe8.png)
在 HTML 和 CSS 里边，如果单词之间没有空格是不会换行的，这样就导致布局被撑大了，出现了空白，解决办法是给email 文本加上 word-break: break-all：
```html
<a data-v-6b416486="" href="mailto:webrequest@salk.edu" style="word-break: break-all;">webrequest@salk.edu</a>
```

**Comment by zqm971210** [comment]
好的 非常感谢  周末休息一了一下没看电脑 非常感谢 up 的帮助
